blueprints/disruption-budgets/README.md

```diff
@@ -0,0 +1,208 @@
+
+## Examples
+The following provides a set of example disruption budgets:
+
+### Only interrupt a % of Karpenter managed nodes
+
+The following Disruption Budgets says, at any-point in time only disrupt 20% of the Nodes managed by the NodePool. For instance, if there were 19 nodes owned by the NodePool, 4 disruptions would be allowed, rounding up from 19 * .2 = 3.8.
+
+```
+apiVersion: karpenter.sh/v1beta1
+kind: NodePool
+metadata:
+  name: default
+spec:
+  ...
+  disruption:
+    consolidationPolicy: WhenUnderutilized
+    expireAfter: 720h
+    budgets:
+    - nodes: "20%"
+```
+
+### Do not disrupt between UTC 09:00 and 18:00 every day
+
+The following Disruption Budgets says, for a 8 hour timeframe from UTC 9:00 don’t disrupt any nodes voluntary as this might be when the application is receiving peak traffic.
+
+```
+apiVersion: karpenter.sh/v1beta1
+kind: NodePool
+metadata:
+  name: default-block-disruptions-during-standard-hours
+spec:
+  ...
+  disruption:
+    consolidationPolicy: WhenUnderutilized
+    expireAfter: 720h
+    budgets:
+    - nodes: "0"
+      schedule: "0 9 * * *"
+      duration: 8h
+```
+
+### Allow 20% disruptions during a maintenance window from UTC 22:00 to 2:00, but only 10% disruptions outside of a maintenance window
+
+The following Disruption Budgets says, for a 4 hour timeframe from UTC 22:00 only disrupt 20% of nodes, but during normal operations only disrupt 10% of nodes.
+
+```
+apiVersion: karpenter.sh/v1beta1
+kind: NodePool
+metadata:
+  name: default-block-disruptions-during-maintenance-window
+spec:
+  ...
+  disruption:
+    consolidationPolicy: WhenUnderutilized
+    expireAfter: 720h
+    budgets:
+    - nodes: "20%"
+      schedule: "0 22 * * *"
+      duration: 4h
+    - nodes: "10%"
+```
+
+### Multiple budgets defined
+
+The following Disruption Budgets is an example of a NodePool with three budgets defined.
+
+* The first budget will only allow 20% of nodes owned by that NodePool to be disrupted.
+* The second budget acts as a ceiling to the previous budget, only allowing 5 disruptions
+* The last budget only blocks disruptions during the first 10 minutes of the day, where 0 disruptions are allowed.
+
+If multiple Budgets are active at the same time Karpenter will consider the most restrictive. You might consider multiple disruption budgets if you want to have a default disruption policy and would like an alternative policy at a specific time e.g. during maintenance windows allow more disruptions to roll-out new Amazon Machine Images faster.
+
+```
+apiVersion: karpenter.sh/v1beta1
+kind: NodePool
+metadata:
+  name: default
+spec:
+  ...
+  disruption:
+    consolidationPolicy: WhenUnderutilized
+    expireAfter: 720h # 30 * 24h = 720h
+    budgets:
+    - nodes: "20%"
+    - nodes: "5"
```

Comment:
We should explain that double entry on node added here and how this applies within the same schedule..

Reply:
Thank you, added that even though we can disrupt 20% of nodes, as both budgets are active karpenter will honor the most restrictive which means only a maximum of 5 nodes at any time.